src/index.ts

```diff
@@ -31,39 +39,73 @@ export class TestxServer {
   private queries: { [id: string]: any };
   private mutations: { [id: string]: any };
 
+  /**
+   * Create a TestxServer.
+   * @constructor
+   * @param {String} schema - The GraphQL data model
```

Comment:
```suggestion
   * @param {string} schema - The GraphQL data model that will be parsed by Graphback
```
or
```suggestion
   * @param {string} schema - The Grahpback data model definition 
   * @see {@link https://graphback.dev/docs/datamodel|Grahpback data model definition}
```
another way will be to add the link as part of the class documentation so that here is enough to say
```suggestion
   * @param {string} schema - The Grahpback data model definition 
```
the link is quite useful especially because if you google for it you will not find anything